examples/capture/main.rs

```diff
@@ -24,22 +24,25 @@ async fn run(png_output_path: &str) {
     create_png(png_output_path, device, buffer, &buffer_dimensions).await;
 }
 
-async fn create_red_image_with_dimensions(width: usize, height: usize) -> (Device, Buffer, BufferDimensions) {
+async fn create_red_image_with_dimensions(
```

Comment:
what happened here? the code was not formatted properly?

Reply:
I would imagine, I have my editor set to rustfmt the file when I save, ask it must have been misformatted.